feg/gateway/docker/.env

```
@@ -8,6 +8,8 @@
 COMPOSE_PROJECT_NAME=feg
 DOCKER_REGISTRY=feg_
 IMAGE_VERSION=latest
+MAGMA_GITHUB_URL=https://github.com/facebookincubator/magma.git
```

We use this .env file for local development, so we shouldn't add the changes you've made here. Instead, can you create a new environment file called prod_env which has the changes you have here, and then in the install script, use prod_env. We can then use an environment variable to set the context to either ../../../ or https://github.com/facebookincubator/magma.git#master

I.e in .env:

```suggestion
BUILD_CONTEXT=../../../
```

And in prod_env
```suggestion
BUILD_CONTEXT=https://github.com/facebookincubator/magma.git#master
```